packages/extension-chakra-storefront/src/setup-app.tsx

```diff
@@ -49,6 +52,7 @@ class ChakraStorefront extends ApplicationExtension<Config> {
         return applyHOCs(App, requiredHOCs)
     }
 
+    @CacheResult('_cachedRoutes')
```

Comment:
Did you have time to test adding this decorator in the base class constructor? I know that is one of the options that we talked about.